modules/map/utils/getMapIcon.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  AssistantIcon,
+  BranchIcon,
+  GarageIcon,
+  ParkingLotIcon,
+  ParkomatIcon,
+  PartnerIcon,
+  PPlusRIcon,
+} from '@/assets/map'
```

Comment:
from `@/assests/map/images` ?

Reply:
Oh yes this is an old code, when I was trying to use `.svg`s